hnn_core/network_builder.py

```diff
@@ -278,12 +285,17 @@ def __init__(self, net, trial_idx=0):
 
         self.ncs = dict()
 
+        # if LFP electrodes have been included, we need to calculate
+        # transmembrane currents at each integration step
+        self._expose_imem = True if len(self.net.lfp) > 0 else False
```

Comment:
I'm thinking there may be other reasons you might want to look at membrane currents. Should we add this attribute in `Network` instead of `NetworkBuilder` ? Also maybe call it `record_imem` ?

Reply:
I'm in the same boat here. We currently only expose somatic current recordings, but I totally see a use case for supporting recordings at other locations as well. This already came up in #323 

Not sure I see that path towards an intuitive API for this yet since it will require some familiarity with the cell geometry. But something to keep in the back of our heads.

Reply:
I'm hesitant to expose the currents elsewhere. Note that we don't save the currents; we use them to calculate potentials. For a laminar electrode "shank", this is an order of magnitude less to save compared with all the segment currents!

Actually, @ntolley, I noticed some time ago that the somatic current recordings might not be what you expect? As they are now implemented, they reflect the current passing through GABA (A and B) channels. I'm pretty sure we don't want/need that level of granularity, so I am planning to propose removing them. Somatic _voltages_ make total sense. One could argue the same for dendrites, I suppose, but we could wait for the need to arise there.